test/spec/runners/bls.ex

```diff
@@ -72,8 +72,16 @@ defmodule BLSTestRunner do
     end
   end
 
-  defp assert_aggregate(_input, _output) do
-    assert false
+  defp assert_aggregate(%{signatures: signatures}, output) do
```

Comment:
The error probably comes from here. The `%{signatures: signatures}` match would work for a YAML structure like this:
```yaml
input:
  signatures: []
output: nil
```

but it won't work for something like this:
```yaml
input: []
output: nil
```
(that is, the format that's used)